pygamelib/engine.py

```diff
@@ -3119,3 +3119,63 @@ def display_at(
             eol = self.terminal.clear_eol
         with self.terminal.location(column, row):
             print(text, eol, end=end, file=file, flush=flush)
+    
+    def display_sprite_at(
+        self,
+        sprite,
+        row=0,
+        column=0,
+        file=sys.stdout,
+        flush=False,
+    ):
+        """
+        Displays sprite at a given position.
+
+        :param sprite: The sprite object to display.
+        :type sprite: pygamelib.gfx.core.Sprite
```

Comment:
Could you use the Sphinc referecing system please:
:type sprite: :class:\`~pygamelib.gfx.core.Sprite\`